src/components/modals/CreateNoteModal.jsx

```diff
@@ -1,6 +1,39 @@
+import { useState } from "react";
+import { RichTextEditor } from "../editor/RichTextEditor";
+import { useNotes } from "../../context/notes-context";
 import "./Modals.css";
+import { toast } from "react-toastify";
+
+export function CreateNoteModal({
+  setCreateNoteModalVisible,
+  setIsLabelModalVisible,
+}) {
+  const currentDate = new Date();
+
+  const initialInputData = {
+    title: "",
+    content: "<p><br></p>",
+    isPinned: false,
+    color: "green",
+    tags: [],
+    priority: "low",
+    date: currentDate.toDateString().slice(4),
+    time: currentDate.getHours() + ":" + currentDate.getMinutes(),
```

Comment:
The date is formatting to `22:1` when the time was `22:01`. Can you check into this? Also there is an alternative to this. The backend folder has a `formatDate` function in authUtils. You can use that to show current date and time instead.

Reply:
Thanks for the suggestions.